web/templates/homepage/index.html.eex

```diff
@@ -5,8 +5,8 @@
   </div>
 </section>
 
-<section class="pt1 pb3 pt5-l ph4 ph5-m ph6-l lm-bg-dark-blue tc">
-  <p class="white tl">
+<section class="pt1 pb3 ph4 ph5-m ph6-l lm-bg-dark-blue tc">
+  <p class="white tl tc-l">
```

Comment:
No, text is left on mobile: https://github.com/LDMW/app/issues/77#issuecomment-299870609